.github/workflows/progress-tracker.yml

```diff
@@ -129,6 +131,7 @@ jobs:
                   team.Authors = parseUserNames(authors)
                   team.Reviewers = parseUserNames(reviewers)
                   team.Analysts = parseUserNames(analysts)
+                  team.All = [...team.Authors, ...team.Reviewers, ...team.Analysts]
```

Comment:
I think unique count of total contributors would be more useful. Could do something like `Array.from(new Set([...]))`.

Reply:
I actually wrote the code for making it unique, but reverted it for a few reasons:

* I was not sure how often a single person can be in more than one roles in the same chapter
* The sum can look odd in situations like `1+2+2=3`, but this is a representational thing that can be done differently
* In the Contributor details section, numbers will represent only one entry per chapter

That said, if you think that would be a better representation then I can bring my change back that I put together to make it unique initially.